.existdb.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I've seen it appearing in @wolfgangmm's recent apps, e.g., http://gitlab.exist-db.org/tei-publisher/tei-publisher-app/blob/master/.existdb.json. It's convenient to have here when working on an app with Atom. In hsg-project apps, I have an ant task that generates these `.existdb.json` files - https://github.com/HistoryAtState/hsg-project/blob/master/build.xml#L561-L590 - and I keep the `.existdb.json` files out of source control. We could add a task for generating `.existdb.json` files to the stock `build.xml` file for eXist apps if there's a real problem to keeping these hardcoded files in source control - where modifications to it will be picked up by git. I'd say the approach in this PR isn't ideal. What are your thoughts?